Go 1.15: conversion from untyped int to string yields a string of one rune

Go 1.15 rc 1 on Fedora Rawhide


```
Testing    in: /builddir/build/BUILD/com-1.0.1/_build/src
         PATH: /builddir/build/BUILD/com-1.0.1/_build/bin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/sbin
       GOPATH: /builddir/build/BUILD/com-1.0.1/_build:/usr/share/gocode
  GO111MODULE: off
      command: go test -buildmode pie -compiler gc -ldflags "-X github.com/unknwon/com/version=1.0.1 -extldflags '-Wl,-z,relro -Wl,--as-needed  -Wl,-z,now -specs=/usr/lib/rpm/redhat/redhat-hardened-ld '"
      testing: github.com/unknwon/com
github.com/unknwon/com
# github.com/unknwon/com
./convert.go:26:22: conversion from untyped int to string yields a string of one rune, not a string of digits (did you mean fmt.Sprint(x)?)
./convert.go:159:8: conversion from int to string yields a string of one rune, not a string of digits (did you mean fmt.Sprint(x)?)
./convert.go:161:8: conversion from int to string yields a string of one rune, not a string of digits (did you mean fmt.Sprint(x)?)
FAIL	github.com/unknwon/com [build failed]
```

See https://github.com/golang/go/issues/32479

--- a/convert.go
+++ b/convert.go
@@ -23,7 +23,7 @@ import (
 type StrTo string
 
 func (f StrTo) Exist() bool {
-	return string(f) != string(0x1E)
+	return string(f) != string(rune(0x1E))
 }
 
 func (f StrTo) Uint8() (uint8, error) {
@@ -156,9 +156,9 @@ func Int2HexStr(num int) (hex string) {
 
 		c := "?"
 		if r >= 0 && r <= 9 {
-			c = string(r + '0')
+			c = string(rune(r + '0'))
 		} else {
-			c = string(r + 'a' - 10)
+			c = string(rune(r + 'a' - 10))
 		}
 		hex = c + hex
 		num = num / 16

